### PR TITLE
[fixes #2143] Add next round of experiments pages (rough versions)

### LINKED ISSUE #2143: Add next round of Test Pilot experiments to testpilot dev server
It'd be nice to get the signed XPIs for the following experiments onto the dev server for QA testing sooner rather than later:

- [Containers](https://github.com/mozilla/testpilot-containers)
- [Snooze Tabs](https://github.com/bwinton/SnoozeTabs) &mdash; see https://testpilot.firefox.com/files/snoozetabs@mozilla/latest
- [Pulse](https://github.com/mozilla/pulse)

We've tested these somewhat thoroughly, but independently. It'd be nice to be able to see how/if they work together (as well as with previous experiments). For example, https://github.com/bwinton/SnoozeTabs/issues/151.

Even if we just have a bunch of Lorem Ipsum and dummy content in the YAMLs, giving localizers and QA at least 1-2 weeks to test on testpilot.dev.mozaws.net would be super helpful.

### PR DESCRIPTION
**Pull request by ckprice**
- [x] Snooze Tabs product page
- [x] Pulse product page
- [x] Containers product page

Fixes #2143.

### CODE REVIEW

**Comment by lmorchard** [comment]
FWIW, could be worth splitting this up into PRs for each experiment as they're ready. (he says selfishly on behalf of snooze tabs)

**Comment by ckprice** [comment]
Note: The purpose of this PR is to get something functional up that has XPI URL's for QA. There will be a follow-up PR which addresses copy/asset polishing

**Comment by ckprice** [comment]
Pulse product page added with a placeholder XPI URL (empty now, but it represents the final location).

**Comment by pdehaan** [comment]
Not sure if we have a standard `contributors.avatar[]` size for images (seems to either be 64x64 of 288x288).
Plus, not sure if we can optimize the `details.images[]`, as 740KB each is a bit heavy.

➜  testpilot git:(issue-2143-next-round) ✗ node imglint.js pulse snooze-tabs

### ./content-src/experiments/pulse.yaml
| KEY | SIZE | DIMENSIONS | PATH
|-----|-----:|-----------:|-----
| thumbnail | 5.1KB | 192x192 | ./frontend/src/images/experiments/pulse/icon/thumbnail.png
| image_twitter | 39.3KB | 561x300 | ./frontend/src/images/experiments/pulse/social/pulse-twitter.jpg
| image_facebook | 111.8KB | 1280x630 | ./frontend/src/images/experiments/pulse/social/pulse-facebook.jpg
| contributors.avatar[0] | 2.3KB | 64x64 | ./frontend/src/images/experiments/universal-search/avatars/chuck-harmston.jpg
| contributors.avatar[1] | 73.9KB | 288x288 | ./frontend/src/images/experiments/avatars/sevaan.jpg
| contributors.avatar[2] | 60.1KB | 288x288 | ./frontend/src/images/experiments/avatars/harald.jpg
| contributors.avatar[3] | 46.7KB | 288x288 | ./frontend/src/images/experiments/avatars/peter.jpg
| details.image[0] | 739.1KB | 850x640 | ./frontend/src/images/experiments/pulse/details/detail1.png
| details.image[1] | 748.3KB | 850x640 | ./frontend/src/images/experiments/pulse/details/detail2.png
| tour_steps.image[0] | 54.8KB | 1280x720 | ./frontend/src/images/experiments/pulse/tour/tour1.jpg
| tour_steps.image[1] | 73.3KB | 1280x720 | ./frontend/src/images/experiments/pulse/tour/tour2.jpg
| tour_steps.image[2] | 70.1KB | 1280x720 | ./frontend/src/images/experiments/pulse/tour/tour3.jpg
| tour_steps.image[3] | 58.1KB | 1280x720 | ./frontend/src/images/experiments/pulse/tour/tour4.jpg

### ./content-src/experiments/snooze-tabs.yaml
| KEY | SIZE | DIMENSIONS | PATH
|-----|-----:|-----------:|-----
| thumbnail | 6.7KB | 192x192 | ./frontend/src/images/experiments/snooze-tabs/icon/thumbnail.png
| image_twitter | 30.8KB | 560x300 | ./frontend/src/images/experiments/snooze-tabs/social/snooze-tabs-twitter.jpg
| image_facebook | 83.4KB | 1200x630 | ./frontend/src/images/experiments/snooze-tabs/social/snooze-tabs-facebook.jpg
| contributors.avatar[0] | 69.7KB | 288x288 | ./frontend/src/images/experiments/avatars/les.jpg
| contributors.avatar[1] | 73.9KB | 288x288 | ./frontend/src/images/experiments/avatars/sevaan.jpg
| contributors.avatar[2] | 2.3KB | 64x64 | ./frontend/src/images/experiments/tab-center/avatars/blake-w.jpg
| contributors.avatar[3] | 69.1KB | 400x400 | ./frontend/src/images/experiments/tab-center/avatars/erica-wright.png
| contributors.avatar[4] | 61.3KB | 288x288 | ./frontend/src/images/experiments/avatars/parag.jpg
| contributors.avatar[5] | 46.7KB | 288x288 | ./frontend/src/images/experiments/avatars/peter.jpg
| contributors.avatar[6] | 39.9KB | 288x288 | ./frontend/src/images/experiments/avatars/emil.jpg
| contributors.avatar[7] | 40.9KB | 288x288 | ./frontend/src/images/experiments/avatars/ciprian.jpg
| details.image[0] | 603.2KB | 1280x960 | ./frontend/src/images/experiments/snooze-tabs/details/detail1.jpg
| details.image[1] | 335.4KB | 1200x900 | ./frontend/src/images/experiments/snooze-tabs/details/detail2.jpg
| tour_steps.image[0] | 82.8KB | 1280x720 | ./frontend/src/images/experiments/snooze-tabs/tour/tour1.jpg
| tour_steps.image[1] | 79.3KB | 1280x720 | ./frontend/src/images/experiments/snooze-tabs/tour/tour2.jpg
| tour_steps.image[2] | 76.7KB | 1280x720 | ./frontend/src/images/experiments/snooze-tabs/tour/tour3.jpg
| tour_steps.image[3] | 76.4KB | 1280x720 | ./frontend/src/images/experiments/snooze-tabs/tour/tour4.jpg
| tour_steps.image[4] | 82.6KB | 1280x720 | ./frontend/src/images/experiments/snooze-tabs/tour/tour5.jpg

**Comment by pdehaan** [comment]
Links seem to work, except the 2 discourse links...

```
➜  content-src git:(issue-2143-next-round) ✗ node link-check.js
{ name: 'snooze-tabs',
  images: [],
  urls:
   [ { contentType: 'text/html; charset=utf-8',
       ok: false,
       origUrl: 'https://discourse.mozilla-community.org/c/test-pilot/snooze-tabs',
       status: 404,
       statusText: 'Not Found',
       url: 'https://discourse.mozilla-community.org/c/test-pilot/snooze-tabs' } ] }
{ name: 'min-vid', images: [], urls: [] }
{ name: 'pulse',
  images: [],
  urls:
   [ { contentType: 'text/html; charset=utf-8',
       ok: false,
       origUrl: 'https://discourse.mozilla-community.org/c/test-pilot/pulse',
       status: 404,
       statusText: 'Not Found',
       url: 'https://discourse.mozilla-community.org/c/test-pilot/pulse' } ] }
{ name: 'activity-stream', images: [], urls: [] }
{ name: 'tracking-protection', images: [], urls: [] }
{ name: 'universal-search', images: [], urls: [] }
{ name: 'tab-center', images: [], urls: [] }
{ name: 'cliqz', images: [], urls: [] }
{ name: 'dev-example', images: [], urls: [] }
{ name: 'page-shot', images: [], urls: [] }
{ name: 'no-more-404s', images: [], urls: [] }
```

**Comment by pdehaan** [comment]
> FWIW, could be worth splitting this up into PRs for each experiment as they're ready. (he says selfishly on behalf of snooze tabs)

Since both of these are tagged with `dev: true`, I'm fine with just merging this whenever and doing a separate PR for Containers.

**Comment by ckprice** [comment]
@pdehaan I added a dummy Containers page with the URL @clouserw gave me. I also don't think the XPI has been uploaded to the ~~Snooze Tabs~~ Pulse URL yet (@chuck do you have any idea?).

Either way, maybe we can get this merged in just to play with on dev.

**Comment by pdehaan** [comment]
> I also don't think the XPI has been uploaded to the Snooze Tabs URL yet (@chuck do you have any idea?).

@ckprice: Wait, do you mean Snooze Tabs or Pulse? Because if it's snooze tabs, we should namecheck @bwinton and @lmorchard. If you mean Pulse, then we can namecheck @chuck.

**Comment by ckprice** [comment]
WOOPS sorry, yeah Pulse. Snooze Tabs is all good.

File: content-src/experiments/containers.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    copy: Test.
+  -
+    image: /static/images/experiments/pulse/tour/tour4.jpg
+    copy: Test.
+notifications: []
+contributors:
+  -
+    display_name: 'Sevaan Franks'
+    title: 'Firefox UX'
+    avatar: /static/images/experiments/avatars/generic-profile.png
```
Comment (pdehaan) [inline content-src/experiments/containers.yaml:66]: This is kosher for placeholder (and who doesn't love sevaan), but we should make sure we have titles+avatars ready to go for: https://github.com/mozilla/testpilot-containers/graphs/contributors

At a minimum for contributors, I think we'd need the following people (in no actual order):

- Andrea Marchesini
- Luke Crouch (can steal info from https://testpilot.dev.mozaws.net/experiments/tracking-protection)
- Jonathan Kingston
- Kamil Jozwiak
- Tanvi Vyas
- John Gruen (can steal info from https://testpilot.dev.mozaws.net/experiments/tab-center)

Comment (ckprice) [reply content-src/experiments/containers.yaml:66]: Yeah, these are being addressed in the [experiment copy document](https://docs.google.com/document/d/1Bu7heb5be6qxg754efm7rMpdE_cRVrXiynBNVXXkhII/edit?ts=5899f23c#heading=h.oc2de5wpgi4h). This PR is a rough version just to test XPI installs.

**Comment by ckprice** [comment]
Talked to @pdehaan we're going to merge this just so we have some dev pages up. I'll create a follow-up issue with the items we need for launch.

File: content-src/experiments/containers.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+description: >
+  Contain yourselves! Create different containers for each of your online lives -  
+  your work self, your social self, your personal self - to stay organized and maintain privacy.
+introduction: >
+  Containers let you create profiles in Firefox for all of your online lives. Custom 
+  labels and color-coded tabs help keep different activities  - like online shopping, 
+  travel planning, or checking work email -  separate.  And each container stores cookies 
+  separately, so online trackers can’t connect your browsing in one container to another.  
+  You can keep your shopping self separate from your social self from your work self, 
+  without worrying about being followed around the Web.
```
Comment (TanviHacks) [inline content-src/experiments/containers.yaml:20]: Can we add a sentence about staying logged into the same site with multiple different accounts at the same time?

**Pull request merged**